src/routes/safe/store/models/incomingTransaction.js

```diff
@@ -18,6 +18,27 @@ export type IncomingTransactionProps = {
   executionDate: string,
   type: string,
   status: string,
+  nonce: null,
+  confirmations: null,
+  recipient: null,
+  data: null,
+  operation: null,
```

Comment:
oh why is it there? does our backend model have all these props?

Reply:
Flow was really annoying complaining about that I was trying to access an attribute that is not part of `IncomingTransactionProps` despite it was inside a condition like:

```typescript
const doSomething = (tx: IncomingTransaction | Transaction) => {
  if (tx.type !== INCOMING_TX_TYPE) {
   // access data existing only in `TransactionProps`
  } else {
   //...
  }
```

Probably there's a way to explicitly say that I know I'm dealing with an `Transaction` inside that condition, but I didn't found it and forgot to better look at it after all.